dashboards/pods.libsonnet

```diff
@@ -78,11 +79,23 @@ local numbersinglestat = promgrafonnet.numbersinglestat;
           legend_avg=true,
         )
         .addTarget(prometheus.target(
-          'sort_desc(sum by (pod_name) (rate(container_network_receive_bytes_total{%(cadvisorSelector)s, %(clusterLabel)s="$cluster", namespace="$namespace", pod_name="$pod"}[1m])))' % $._config,
+          'sort_desc(sum by (pod_name) (rate(container_network_receive_bytes_total{%(cadvisorSelector)s, %(clusterLabel)s="$cluster", %(clusterLabel)s="$cluster", namespace="$namespace", pod_name="$pod"}[1m])))' % $._config,
           legendFormat='{{ pod_name }}',
         ))
       );
 
+      local restartAnnotation = annotation.datasource(
+        'Restarts',
+        '$datasource',
+        expr='time() == BOOL timestamp(deriv(kube_pod_container_status_restarts_total{%(kubeStateMetricsSelector)s, %(clusterLabel)s="$cluster", namespace="$namespace", pod="$pod"}[2m]) > 0)',
```

Comment:
I believe `deriv` is only supposed to be used for gauges (does not handle counter resets). Perhaps use `increase` here?

Reply:
You are right.
As increase is only a wrapper to rate, let's use rate.

Beware that it does not correctly all the reboots and seems to duplicate the reboot if the range is too large. irate would be perfect for this job but for some reason I am unable to make it work.